core/blockchain.go

```diff
@@ -2191,6 +2193,35 @@ func (bc *BlockChain) reorg(oldBlock, newBlock *types.Block) error {
 	} else {
 		// len(newChain) == 0 && len(oldChain) > 0
 		// rewind the canonical chain to a lower point.
+
+		home, err := os.UserHomeDir()
+		if err != nil {
+			fmt.Println("Impossible reorg : Unable to get user home dir", "Error", err)
+		}
+		outPath := filepath.Join(home, "impossible-reorgs", fmt.Sprintf("%v-impossibleReorg", time.Now().Format(time.RFC3339)))
+
+		if _, err := os.Stat(outPath); errors.Is(err, os.ErrNotExist) {
+			err := os.MkdirAll(outPath, os.ModePerm)
+			if err != nil {
+				log.Error("Impossible reorg : Unable to create Dir", "Error", err)
+			}
+		}
+
+		err = os.WriteFile(filepath.Join(outPath, "oldchain.txt"), []byte(fmt.Sprintf("%+v", oldChain)), 0600)
```

Comment:
stop. we won't be able to import from that format

Reply:
look for `func ExportChain(blockchain *core.BlockChain, fn string) error` and `func ImportChain(chain *core.BlockChain, fn string) error {` and similar functions in `cmd/utils/cmd.go`

Reply:
Sure, will move on this path going forward.